app/models/project_step.rb

```diff
@@ -194,24 +216,14 @@ def scheduled_bg
     end
   end
 
-  def scheduled_day
-    scheduled_date.day
+  def scheduled_start_day
+    scheduled_start_date.day
   end
 
   # Returns a duplication helper object which encapsulate handling of the modal rendering and
   # submit handling.
   def duplication
-    @duplication ||= ProjectStepDuplication.new(self)
-  end
-
-  def adjust_scheduled_date(days_adjustment)
-    if scheduled_date && days_adjustment != 0
-      new_date = scheduled_date + days_adjustment.days
-      # note, original_date will be assigned if needed by the before_save logic
-      update!(scheduled_date: new_date)
-    else
-      false
-    end
+    @duplication ||= Timeline::StepDuplication.new(self)
```

Comment:
@hooverlunch Seems this is used over here as well [project_steps/_repeat_options.html.slim#L1](https://github.com/sassafrastech/madeline_system/blob/dbf9b2763514cd75a59de2c1aaa8a2db69348c3e/app/views/admin/project_steps/_repeat_options.html.slim#L1)

I could thread it through the various partials, would you still like me to proceed with this change?